docs/ops/activation/GELU_7.md

```diff
@@ -2,53 +2,63 @@
 
 **Versioned name**: *Gelu-7*
 
-**Category**: *Activation*
+**Category**: *Activation function*
 
-**Short description**: Calculates Gaussian error linear.
+**Short description**: Gaussian error linear unit element-wise activation function.
 
-**Detailed description**: `Gelu(x) = x * Φ(x)`, where `Φ(x)` is the Cumulative Distribution Function for Gaussian Distribution.
-The Gelu operation is introduced in the [paper](https://arxiv.org/abs/1606.08415).
+**Detailed description**:
 
-**Attributes**
+*Gelu* operation is introduced in this [article](https://arxiv.org/abs/1606.08415).
+It performs element-wise activation function on a given input tensor, based on the following mathematical formula:
 
-* *approximation_mode*
+\f[
+    Gelu(x) = x\cdot\Phi(x)
+\f]
 
-  * **Description**: Specifies the formulae to calculate the output.
-  * **Range of values**:
-    * `erf` -- calculate output using the Gauss error function.
-    * `tanh` -- calculate output using tanh approximation
-  * **Type**: `string`
-  * **Default value**: `erf`
-  * **Required**: *no*
+where `Φ(x)` is the Cumulative Distribution Function for Gaussian Distribution.
 
+The *Gelu* function may be approximated in two different ways based on *approximation_mode* attribute.
 
-**Mathematical Formulation**
+For `erf` approximation mode, *Gelu* function is represented as:
 
-For the `erf` approximation mode:
 \f[
-    Gelu(x) = 0.5 \cdot x \cdot (1.0 + erf((x) / \sqrt{2})
+    Gelu(x) = x\cdot\Phi(x) = x\cdot\frac{1}{2}\cdot\left[1 + erf\left(x/\sqrt{2}\right)\right]
```

Comment:
![image](https://user-images.githubusercontent.com/20084299/116396441-b17dcc80-a825-11eb-8740-1246f8b4dde7.png)
I would suggest to unify math forms to easier see the difference between `erf` and `tanh` modes.
Is the approximation symbol is relevant only for the `tanh` mode?

Reply:
In the operation [article](https://arxiv.org/pdf/1606.08415.pdf), the approximation formula is strictly for tanh mode. Error function is used to calculate the cumulative distribution function of a Gaussian.

Reply:
Oh okay I see, so probably we should stick with the formulas as they are presented in the paper.